### PR TITLE
Adding HentaiLA presence

### PR DESCRIPTION
**Pull request by u0f**
imagen

imagen

imagen

### CODE REVIEW

**Comment by EncryptedDev** [comment]
@alexbcberio string approval please

File: websites/H/HentaiLA/presence.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    case "/login":
+      presenceData.details = "Inciando sesión";
+      break;
+    case "/registro":
+      presenceData.details = "Registrándose";
+      break;
+  }
+
+  if (!presenceData.details) {
+    presence.setTrayTitle();
```
Comment (EncryptedDev) [inline websites/H/HentaiLA/presence.ts:73]: Function has been deprecated

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/H/HentaiLA/presence.ts (hunk 1 above)
Comment (u0f) [reply websites/H/HentaiLA/presence.ts:73]: What function is now?

Comment (EncryptedDev) [reply websites/H/HentaiLA/presence.ts:73]: Nothing, do you know what it even does? xD

File: websites/H/HentaiLA/presence.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      document.querySelector(".section-title").textContent.split(":")[1]
+    }`;
+  } else if (document.location.pathname.includes("/ver/")) {
+    presenceData.details = "Viendo Hentai:";
+    presenceData.state = document.querySelector(".section-title").textContent;
+    [presenceData.startTimestamp, presenceData.endTimestamp] =
+      presence.getTimestamps(
+        Math.floor(video.currentTime),
+        Math.floor(video.duration)
+      );
```
Comment (Bas950) [inline websites/H/HentaiLA/presence.ts:37]: getMediaTimestamps?

Comment (u0f) [reply websites/H/HentaiLA/presence.ts:37]: imagen
This function?

Comment (EncryptedDev) [reply websites/H/HentaiLA/presence.ts:37]: getTimestampsFromMedia

Comment (u0f) [reply websites/H/HentaiLA/presence.ts:37]: I think "getTimestamps" better to use and more easly 🤔.
It works perfect 😞💥

Comment (EncryptedDev) [reply websites/H/HentaiLA/presence.ts:37]: No, use getTimestampsFromMedia()

Comment (u0f) [reply websites/H/HentaiLA/presence.ts:37]: How it works

Comment (EncryptedDev) [reply websites/H/HentaiLA/presence.ts:37]: https://docs.premid.app/en/dev/presence/class#gettimestampsfrommediahtmlmediaelement

Comment (Slowlife01) [reply websites/H/HentaiLA/presence.ts:37]: *Video is not videoelement.*Literally an object with duration, currently time and paused

**Review by Slowlife01 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**